programming/sublime-text/actions.py

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/python
+
+# Created For Solus Operating System
+
+from pisi.actionsapi import get, pisitools, shelltools
+import shutil
+
+def setup():
+    shelltools.system("tar xf sublime_text_build_4113_x64.tar.xz")
```

Comment:
You should probably use the version directly, similar to how it was before. So we/you don't have to change that string with every update.

`shelltools.system("tar xf sublime-text_build-%s_x64.tar.xz" % (get.srcVERSION()))`

Reply:
Good point, I was wondering about this. When I was maintainer of Fedy, We used shell commands `cut, sed, etc` to remove hardcoded versions. I attempted similar approach here but didn't knew about this inbuilt command.